src/pages/data-visualization/overview.mdx

```diff
@@ -163,19 +141,9 @@ Just as the core values of IBM have remained consistent over the years, similarl
 
 <Row>
 <Column colMd={5} colLg={8}>
-Successful systems require both ﬁxed and fluid elements of expression. The innovative ways in which we use these elements deliver distinction and uniqueness, but the context should be the primary driver of our decisions. A visualization should be crafted based on the environment of the audience and its constraints. If it’s for marketing or analysis purposes, what’s the level of familiarity with the subject matter, and does the ﬁnal result respect the time required by the audience being asked to experience it? These examples are just a few of the essential aspects in a data visualization that affect layout and style, the level of complexity, and the amount of detail revealed.
-</Column>
-<Column colMd={2} colLg={3} >
-
-<div className="mini-card-container">
-
-<p className="description">Learn more on Carbon.</p>
-
-<MiniCard link="https://www.carbondesignsystem.com/data-visualization/getting-started/vanilla" labelName="Data visualization" iconImage={favicon} altText="carbon"></MiniCard>
-
-
-</div>
 
+Successful systems require both ﬁxed and fluid elements of expression. The innovative ways in which we use these elements deliver distinction and uniqueness, but the context should be the primary driver of our decisions. A visualization should be crafted based on the environment of the audience and its constraints. If it’s for marketing or analysis purposes, what’s the level of familiarity with the subject matter, and does the ﬁnal result respect the time required by the audience being asked to experience it? These examples are just a few of the essential aspects in a data visualization that affect layout and style, the level of complexity, and the amount of detail revealed. [Learn more on Carbon.](https://www.carbondesignsystem.com/data-visualization/getting-started)
```

Comment:
```suggestion
Successful systems require both ﬁxed and fluid elements of expression. The innovative ways in which we use these elements deliver distinction and uniqueness, but the context should be the primary driver of our decisions. A visualization should be crafted based on the environment of the audience and its constraints. If it’s for marketing or analysis purposes, what’s the level of familiarity with the subject matter, and does the ﬁnal result respect the time required by the audience being asked to experience it? These examples are just a few of the essential aspects in a data visualization that affect layout and style, the level of complexity, and the amount of detail revealed. [Learn more on Carbon](https://www.carbondesignsystem.com/data-visualization/getting-started).
```

Reply:
What changed here? The diff makes it tough to tell.